rust/ql/lib/codeql/rust/dataflow/internal/SsaImpl.qll

```diff
@@ -426,3 +459,44 @@ class PhiReadNode extends DefinitionExt, Impl::PhiReadNode {
 
   override Location getLocation() { result = Impl::PhiReadNode.super.getLocation() }
 }
+
+private module DataFlowIntegrationInput implements Impl::DataFlowIntegrationInputSig {
+  class Expr extends CfgNodes::ExprCfgNode {
```

Comment:
I think that we want to include patterns for Rust, so this should perhaps just extends `AstCfgNode`.

Reply:
That makes sense. Done!